src/api/cvc4cpp.cpp

```diff
@@ -2077,7 +2095,6 @@ Term Solver::mkUniverseSet(Sort sort) const
     CVC4_API_ARG_CHECK_EXPECTED(!sort.isNull(), sort) << "non-null sort";
     Term res =
         d_exprMgr->mkNullaryOperator(*sort.d_type, CVC4::kind::UNIVERSE_SET);
-    (void)res.d_expr->getType(true); /* kick off type checking */
```

Comment:
I'd rather have this not removed if this is undesired behavior. Disable and assign an issue. We can fix this in a separate PR.